grabbit/src/main/groovy/com/twcable/grabbit/client/batch/ClientBatchJobExecutionListener.groovy

```diff
@@ -121,10 +124,16 @@ class ClientBatchJobExecutionListener implements JobExecutionListener {
 
         final String encodedContentAfterDate = URLEncoder.encode(contentAfterDate, 'utf-8')
         final String encodedPath = URLEncoder.encode(path, 'utf-8')
-        final grabPath = "http://${host}:${port}/grabbit/job?path=${encodedPath}&after=${encodedContentAfterDate}"
+
+        URIBuilder uriBuilder = new URIBuilder(scheme: "http", host: host, port: port as Integer, path: "/grabbit/job")
+        uriBuilder.addParameter("path", encodedPath)
+        uriBuilder.addParameter("after", encodedContentAfterDate)
+        for(String excludePath : excludePaths) {
+            uriBuilder.addParameter("excludePath", URLEncoder.encode(excludePath, 'UTF-8'))
+        }
```

Comment:
@jdigger @masroormohammed this for loop will basically add ... `?excludedPath=a&excludedPath=b&excludedPath=c` and so on. right? At first this seemed odd, but then I saw `GrabbitServlet` has `request.getParameterValues()` which then makes sense.

Just wanted to confirm ...